.github/workflows/node-ci.yml

```diff
@@ -31,7 +31,6 @@ jobs:
               id: match
               with:
                   changed-files: ${{ steps.changed.outputs.files }}
-                  files: packages/ # Only look for changes in packages
```

Comment:
This is wrong. We _do_ want to look for changes in all workspace packages (`config/build` for example, should be checked).